packages/ember-glimmer/lib/syntax/outlet.js

```diff
@@ -201,7 +201,7 @@ const TOP_LEVEL_MANAGER = new TopLevelOutletComponentManager();
 
 class OutletComponentManager extends AbstractOutletComponentManager {
   create(definition, args, dynamicScope) {
-    let outletStateReference = dynamicScope.outletState = dynamicScope.outletState.get(definition.outletName);
+    let outletStateReference = dynamicScope.outletState = dynamicScope.outletState.get('outlets').get(definition.outletName);
```

Comment:
This change feels a bit dubious to me – the reason for this change is that you could swap in the outlet state reference for a regular `EmberPathReference` in your code. However, we are still using the `ChildOutletStateReference` by default, which doesn't fully behave or duck-type like a regular path reference. For example, it has a `get isTopLevel` on it, which your `EmberPathReference` is not going to have. (I _think_ it happened to be unused, but we should just remove it if true.)

If we need to support replacing this with a `EmberPathReference`, I think we need to actually use that ourselves to guarantee compatibility. At minimum, we need the top-level `OutletStateReference` to return a `EmberPathReference` for its `get`, which would go through the regular paths from there onwards.